html-relative-links/html-relative-links/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+   <li><a href="index.html">Home Page</a></li>
+    <li><a href="../html-relative-links/about-me/about-me.html">About Me</a></li>
+    <li><a href="../html-relative-links/projects/projects.html">Projects</a></li>
+    <li><a href="../html-relative-links/contact-us/contact-us.html">Contact Us</a></li>
```

Comment:
Here the file is already inside the `html-relative-links` folder so you don't wan't to leave it then come back in. You can just specify the folder and the file, for example:

```HTML
<li><a href="about-me/about-me.html">About Me</a></li>
```